R/exceptions.R

```diff
@@ -20,24 +20,24 @@ EXCEPTION_TABLE <- tibble::tribble(
-  "22", "Error reading sheet in raw data file", "Error reading %s, sheet %s", "error", "raw data",
+  "22", "Error reading sheet in the raw data file", "Error reading %s, sheet %s", "error", "raw data",
   "23", "Error with readout values in plates", "In result file %s (sheet %s) readout values are misplaced for plate %s", "error", "raw data",
   "24", "Error with template headers", "Template does not contain all expected headers for a '%s'. '%s' is/are required. Please correct your template or change the identifiers.", "error", "template",
   "25", "Error with template file sheets", "Template file %s does not contain %s sheets. Please correct your template.", "error", "template",
   "26", "Metadata special characters", "Metadata field names for %s cannot contain special characters or start with a number: %s", "error", "metadata",
   "27", "Metadata invalid", "Metadata field name: %s in %s is not valid (reserved for output)", "error", "metadata",
-  "28", "Raw Data delimiter", "Can't guess separator fo the delimited files: %s", "error", "raw data",
+  "28", "Raw Data delimiter", "Can't guess separator for the delimited files: %s", "error", "raw data",
   "29", "Raw Data EnVision", "Error reading %s: not an original EnVision .csv file", "error", "raw data",
   "30", "Raw Data Plate size", "Error reading %s: wrong plate size", "error", "raw data",
   "31", "Raw Data structure", "In result file %s (sheet %s) readout values are misplaced for plate %s.", "error", "raw data",
-  "32", "Template File Gnumber and Concentratoin", "Template file(s) %s do/does not contain the same number of Gnumber_* and Concentration_* sheets. Gnumber_* and Concentration_* sheets are required. Please correct your template.", "error", "template",
+  "32", "Template File Gnumber and Concentration", "Template file(s) %s do/does not contain the same number of Gnumber_* and Concentration_* sheets. Gnumber_* and Concentration_* sheets are required. Please correct your template.", "error", "template",
   "33", "Error loading input files", "Some input files are missing. Please upload only files with names starting with Manifest_*, Template_* and RawData_*", "error", "input files",
   "34", "Missing drug annotation", "The following drugs IDs are missing: '%s'. Don't worry, drug names will be used for them.", "warning", "submit tab",
   "35", "Missing cell line annotatation", "The following cell line IDs are missing: '%s'. Don't worry, cell line names will be used for them.", "warning", "submit tab",
-  "36", "Invalid averaged data", "Averaged dose-reponse data for the selected cell line and drug:'%s' is invalid. Please contanct gdrplatform team via 'gdr-support-d@gene.com'.", "error", "assay data"
+  "36", "Invalid averaged data", "Averaged dose-reponse data for the selected cell line and drug: '%s' can be considered invalid. Please check your data in the module 'Manage Data' or contact gdrplatform team via 'gdr-support-d@gene.com'.", "error", "assay data"
```

Comment:
Or maybe:
""Invalid averaged data", "Averaged dose-reponse data for the selected cell line and drug: '%s' will be considered invalid..."